### DISCUSSION TITLE
Interface with Localite

### DISCUSSION THREAD

**Discussion by JonteP** [umair-hassan/best-toolbox#42] [Q&A]
From the publication in Brain Stimulation, I got the impression that BEST has built in support for interfacing with neuronavigation equipment such as Localite. However, I cannot find any documentation on this. Any clues on how to move forward with this is greatly appreciated. Specifically, we want to read current coil coordinates.

**Comment by umair-hassan** [comment]
Hi Jonte,

Thank you for your question. Localite does provide the coil coordinates to the toolbox in an under development version of their navigator software. We are collaborating with them to make this feature exclusively available to all users and on the same grounds we are working with InVesalius Navigator. Can you elaborate your use case a bit more? I may be able to answer you a bit more technically. Do you want to read the coil coordinates in real-time? Or post-hoc is also okay?

Best, Umair

**Reply by JonteP** [reply]
Thank you for a quick and helpful reply! Sounds like exciting features ahead for the toolbox.
What we want to do are basically two related things, one is to record the coil coordinates each time a TMS pulse/trigger is delivered and store with its associated neurophys responses of interest. The other would be to detect whenever the coil is at a target coordinate (within some margin of error). I guess the latter would require real-time, but it is also less important if difficult to solve.

Best,
Jonas

**Reply by umair-hassan** [reply]
Hi Jonas,

Thank you for putting this on the discussion forum here. This will motivate me. I am almost done with API of real-time neuronavigation coordinates. Most likely you can hear the news from my twitter (twitter.com/neuro_engineer) or from the new release of this repository. 

Talk soon,
Umair

**Reply by JonteP** [reply]
Thank you, I look forward to updates on this topic!
